dt-contacts/contacts-post-type.php

```diff
@@ -702,6 +703,12 @@ public function get_custom_fields_settings( $include_current_post = true, int $p
         }
         $fields = $this->get_contact_field_defaults( $post_id, $include_current_post );
         $fields = apply_filters( 'dt_custom_fields_settings', $fields, "contacts" );
+
+        $langs = dt_get_available_languages();
+        array_push( $langs, array(
```

Comment:
Should we add this directly to the dt_get_available languages function?

Reply:
I considered this but didn’t know how that might things that are already using this function.

Reply:
It is a newer function and looks like it is only being used in the people groups translation feature you did earlier. We might test that and give it the go ahead.

Reply:
great. I didn't really take the time to investigate. I will make the change.